tools/update.py

```diff
@@ -205,6 +207,29 @@ def update_karamel(new_dist_dir):
     shutil.copytree(os.path.join(new_dist_dir, "karamel/krmllib"), "karamel/krmllib")
 
 
+def update_vale(new_dist_dir):
+    new_dist_dir = Path(new_dist_dir)
+
+    rm("vale/include")
```

Comment:
This should be made Windows compatible at some point. But it's fine for now as the entire update process isn't right now.